benchmark/atoi.cpp

```diff
@@ -23,7 +29,7 @@ uint32_t parse_eight_digits_swar(const char *chars) {
 
 // Note: eight digits can represent from 0 to (10^9) - 1, the logarithm base 2
 // of 10^9 is slightly less than 30, thus, only 30 bits are needed.
-auto lemire_as_zoo_swar(const char *chars) {
+uint32_t lemire_as_zoo_swar(const char *chars) {
```

Comment:
Truly astonishing to see you return a concrete type. :D

Reply:
It is an exported symbol, I have to manually set the type in the header, I may as well set it manually in the implementation.  Good to see that you noticed